### PR TITLE
Switch org complete implementation

### PR DESCRIPTION
**Pull request by Julias0**

### CODE REVIEW

File: src/app/auth/switch-org/switch-org.page.html [hunk 1]
```diff
@@ -1,20 +1,61 @@
...
     <mat-form-field class="switch-org--form-field" appearance="outline">
-        <span class="material-icons switch-org--search-icon" matPrefix>
-            search
-        </span>
-      <input id="switch-org--search-input" matInput placeholder="Placeholder">
-    </mat-form-field> 
+      <span class="material-icons switch-org--search-icon" matPrefix>
+        search
+      </span>
+      <input #searchOrgsInput id="switch-org--search-input" matInput placeholder="Placeholder">
```
Comment (iamtarun1993) [inline src/app/auth/switch-org/switch-org.page.html:7]: what is `#searchOrgsInput`

File: src/app/auth/switch-org/switch-org.page.ts [hunk 2]
```diff
@@ -1,26 +1,184 @@
...
+    private authService: AuthService,
+    private storageService: StorageService,
+    private router: Router,
+    private networkService: NetworkService,
+    private orgService: OrgService,
+    private userEventService: UserEventService
   ) { }
 
   ngOnInit() {
-    this.offlineService.load().pipe(tap(console.log)).subscribe(noop);
```
Comment (iamtarun1993) [inline src/app/auth/switch-org/switch-org.page.ts:23]: remove console.log if not needed...

File: src/app/auth/switch-org/switch-org.page.scss [hunk 3]
```diff
@@ -1,11 +1,56 @@
...
+      border: none;
+    }
+  }
+
+  &--org-element {
+    padding: 4px;
+    border-bottom: 1px solid $header-border;
+    &-name {
+      font-size: 16px;
+      color: #333;
```
Comment (iamtarun1993) [inline src/app/auth/switch-org/switch-org.page.scss:48]: variable sir..

File: src/app/auth/switch-org/switch-org.page.scss [hunk 4]
```diff
@@ -1,11 +1,56 @@
...
+    }
+
+    &-domain {
+      font-size: 12px;
+      color: #666;
```
Comment (iamtarun1993) [inline src/app/auth/switch-org/switch-org.page.scss:53]: variable

File: src/app/auth/switch-org/switch-org.page.ts (hunk 2 above)
Comment (Julias0) [reply src/app/auth/switch-org/switch-org.page.ts:23]: ei this is a removed comment

**Review by iamtarun1993 — APPROVED** [review]

**Pull request closed without merging**